### PR TITLE
Assistant PDAs start with the bounty board app preinstalled

### PR DESCRIPTION
**Pull request by Mickyan**
## About The Pull Request

Makes assistant pdas start with the bounty board app preinstalled

## Why It's Good For The Game
The bounty board is stuck in a vicious cycle where it sits unused because it lacks visibility, and in turn nobody uses it because posting bounties is like screaming into the void. I believe that by knowing for certain that all assistants on station will hear about your bounty this feature will see much more regular use

Secondly, I believe this is exactly the kind of thing that the assistant job should be about: doing odd jobs, answer to a recruitment call from a department head of staff, and so on. We can't force players to do it but we can guide them towards it. By having the app preinstalled, assistants will be naturally more inclined to check for bounties as they are posted.

I believe solving both of these issues has the potential to greatly increase interactions between the crew while giving helpful assistants a much easier time in finding people that need help.

## Changelog

:cl:
qol: Assistant PDAs now start with the bounty board app preinstalled
/:cl:

### CODE REVIEW

**Comment by Mickyan** [comment]
I would also be interested in expanding this to all crew if maintainers think it's a good idea, I have some mixed feelings about it but more good than bad

**Comment by the-orange-cow** [comment]
> I would also be interested in expanding this to all crew if maintainers think it's a good idea, I have some mixed feelings about it but more good than bad

Might be irritating.

**Comment by JohnFulpWillard** [comment]
I think it's fine as assistant-only, fits in with their job of helping others.

**Comment by JohnFulpWillard** [comment]
i updated your changelog's tag to qol

File: code/modules/modular_computers/computers/item/role_tablet_presets.dm [hunk 1]
```diff
@@ -371,6 +371,12 @@
  * Non-roles
  */
 
+/obj/item/modular_computer/tablet/pda/assistant
```
Comment (JohnFulpWillard) [inline code/modules/modular_computers/computers/item/role_tablet_presets.dm:374]: Assistant is a role so shouldn't be in the non-roles section, make a 'no department' section or just put them in service, either is fine.

**Review by JohnFulpWillard — CHANGES_REQUESTED** [review]

File: code/modules/modular_computers/computers/item/role_tablet_presets.dm (hunk 1 above)
Comment (Mickyan) [reply code/modules/modular_computers/computers/item/role_tablet_presets.dm:374]: Ding dang done

**Pull request merged**